TEAM.md

```diff
@@ -4,12 +4,14 @@ These are the general guidelines for maintaining WHATWG standards. Mostly boring
 
 ## Handling pull requests
 
-Rules for title/message must be satisfied before pull request is reviewed. See [COMMITTING.md](COMMITTING.md) for further details.
+Rules for title/message must be satisfied before pull request (PR) is reviewed. See [COMMITTING.md](COMMITTING.md) for further details.
 
 For normative changes, ask for a [web-platform-tests](https://github.com/w3c/web-platform-tests) PR if testing is practical and not overly burdensome. Aim to merge both PRs at the same time. If one PR is approved but the other needs more work, add the `do not merge yet` label or, in web-platform-tests, the `status:needs-spec-decision` label.
 
 If a follow-up issue is filed in the web-platform-tests repository, add the `type:untestable` or `type:missing-coverage` label, and any other appropriate labels, e.g. `html` for the HTML Standard.
 
+Furthermore, for changes affecting one or more implementations, ensure implementation bugs are filed.
```

Comment:
Should we give some links to the bug trackers, similar to https://github.com/heycam/webidl/blob/master/README.md#filing-issues-elsewhere ? Component-less, though.

Reply:
Done.